frontend/public/components/image-stream-tag.tsx

```diff
@@ -55,9 +55,9 @@ export const ImageStreamTagsDetails: React.SFC<ImageStreamTagsDetailsProps> = ({
       <div className="co-m-pane__body-group">
         <div className="row">
           <div className="col-md-6 col-sm-12">
-            <SectionHeading text="Image Details" />
+            <SectionHeading text="ImageStreamTag details" />
```

Comment:
@yapei Do you mind opening a bug to translate these properties? We can do it as a separate bug. It looks like we missed these.

Reply:
@spadgett  I opened [one](https://bugzilla.redhat.com/show_bug.cgi?id=1960102)